WebApp/autoreduce_webapp/reduction_variables/views.py

```diff
@@ -347,19 +346,27 @@ def run_confirmation(request, instrument=None):
         context_dictionary['error'] = 'Runs span multiple experiment numbers (' + ','.join(str(i) for i in rb_number) + ') please select a different range.'
 
     for run_number in run_numbers:
-        old_reduction_run = ReductionRun.objects.filter(run_number=run_number).order_by('-run_version').first()
+        matching_previous_runs_queryset = ReductionRun.objects.filter(instrument=instrument,
+                                                                      run_number=run_number).order_by('-run_version')
+        most_recent_previous_run = matching_previous_runs_queryset.first()
 
         # Check old run exists
-        if old_reduction_run is None:
+        if most_recent_previous_run is None:
             context_dictionary['error'] = "Run number " + str(run_number) + " doesn't exist."
```

Comment:
I'd change this from `doesn't exist.` to something like ` hasn't been ran by autoreduction, can't rerun.`. Just so it's clearer why the user can't submit a job.